crates/autopilot/src/driver_model.rs

```diff
@@ -150,6 +151,16 @@ pub mod solve {
         pub submission_address: H160,
     }
 
+    #[serde_as]
+    #[derive(Copy, Clone, Debug, Default, Serialize, Deserialize)]
+    #[serde(rename_all = "camelCase")]
+    pub struct ProtocolFee {
```

Comment:
This should already be designed for the future (to avoid breaking API changes) as an enum and PriceImprovement being the first subtype. Also please document the what the type of fee means.

Reply:
Renamed and converted to enum.